browser/src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { NavigationMenu } from './view/components/Navbar/NavigationMenu';
+
+export const App = () => {
+  return (
+    <>
+      <NavigationMenu />
+    </>
```

Comment:
You only need to use fragments if:

1) you have more than one element without a wrapper
2) you don't wrap your "more than one element without a wrapper" in an array